README.md

```diff
@@ -49,33 +49,38 @@ On the endpoint, JSON will be POSTed with the following structure:
-| WEBHOOK_ABORT_TIMEOUT_MS         | No       | 10_000                 | Timeout for webhook response                                                                                                                                                                                                  |
-| WEBHOOK_MAX_RETRIES         | No       | 0                 | Maximum retries for webhook                                                                                                                                                                                                  |
-| CAPELLA_FORK_VERSION        | No       | 0x03000000        | Optional parameter to specify Capella fork version                                                                                                                                                                           |
+| Variable                                      | Required | Default/Example       | Description                                                                                                                                                                                                                                             |
+|-----------------------------------------------|----------|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| EXECUTION_NODE                                | Yes      | http://1.2.3.4:8545   | Ethereum Execution Node endpoint                                                                                                                                                                                                                        |
+| CONSENSUS_NODE                                | Yes      | http://1.2.3.4:5051   | Ethereum Consensus Node endpoint                                                                                                                                                                                                                        |
+| JWT_SECRET_PATH                               | No       | /data/neth/jwt.hex    | Path to JWT secret hex file for Nethermind RPC authentication. If provided, will generate a new JWT token for each execution node request                                                                                                               |
+| LOCATOR_ADDRESS                               | Yes      | 0x123                 | Address of the Locator contract [Goerli](https://docs.lido.fi/deployed-contracts/goerli/) / [Mainnet](https://docs.lido.fi/deployed-contracts/)                                                                                                         |
```

Comment:
let's update the Goerli part here and replace it with the actual locator from Hoodi?